src/screens/DiscoveryScreenV2/elements/DiscoveryTab.js

```diff
@@ -67,7 +67,7 @@ const DiscoveryTab = ({onChangeScreen, selectedScreen = 0, tabs}) => {
 const styles = StyleSheet.create({
   tabContainer: {
     height: dimen.normalizeDimen(38),
-    backgroundColor: colors.white
+    backgroundColor: COLORS.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color of the tab container has been updated from `colors.white` to `COLORS.white`. This change is consistent with the update to the import statement and should not cause any issues.
```diff
- backgroundColor: colors.white
+ backgroundColor: COLORS.white
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->